bots/psuteam7bot/prophet.js

```diff
@@ -28,20 +28,27 @@ prophet.doAction = (self) => {
         if(self.isRadioing(baseRobot))  //Check if base has broadcasted a signal on it's turn
         {
             //Get the message using baseRobot.signal and translate to position using helper function
-            self.potentialEnemyCastleLocation = [communication.signalToPosition(baseRobot.signal, self.map)];
-            self.potentialEnemyCastleLocation.push(movement.getDiagonalPatrolPosition(self.base, self.map));
```

Comment:
This test is likely what is making all those tests fail. Maybe leave this in and just add the prophet setting? Just in case we need attacker/defender prophets or something?

Reply:
Or if you refactored it out completely that's fine too. Not sure how much you changed/removed

Reply:
I changed the tests to use `self.target`, have a bigger map and farther target so it does not "report" destroyed enemy castle and set `self.target` to null